### ISSUE TITLE
Random Seed Setting

### ISSUE THREAD

**Issue by acmater** [saratanov/git_honours#3]
I figured we should practice with all features of GitHub, so I thought I'd ask my question here instead of via email.

I haven't looked closely enough at the whole code to know this, but are random seeds set for numpy, python, and torch at some point? If not, we should redo these calculations after Honours with all of these seeds fixed so that it is reproducible.

**Comment by saratanov** [comment]
I haven't set random seeds for numpy, python, torch, but I have set the seeds for the sklearn train/test split functions. Probably too late to rerun everything now but I will add seeds after it's all done.

**Comment by acmater** [comment]
Yeah, we just want to make sure we do it with seeds before submitting the paper. Definitely all good for Honours.
